submit_and_compare/submit_and_compare.py

```diff
@@ -1,34 +1,32 @@
-'''  Submit and Compare XBlock main Python class'''
+"""  Submit and Compare XBlock main Python class """
 
+import StringIO
+import textwrap
+import logging
 import pkg_resources
+
 from django.template import Context, Template
 from django.utils.translation import ungettext
-
 from xblock.core import XBlock
 from xblock.fields import Scope, String, List, Float, Integer
 from xblock.fragment import Fragment
-
 from lxml import etree
-from xml.etree import ElementTree as ET
 
-from StringIO import StringIO
-
-import textwrap
-
-import logging
-
-log = logging.getLogger(__name__)
+LOG = logging.getLogger(__name__)
```

Comment:
That is standard pylint.  Any variables outside the class definition are considered constants and must be caps.